Add missing KIC 3.0 docs (#6530)
Kong/docs.konghq.com: app/_src/kubernetes-ingress-controller/concepts/gateway-api.md

Gateway API and Kong's implementation of Gateway API are both Generally Available for all users.

{:.note}

> > Gateway API resources will only be reconciled when the Gateway API CRDs are installed in your cluster _before_ {{ site.kic_product_name }} is started. See the [getting started](/kubernetes-ingress-controller/{{ page.release }}/get-started/) page for installation instructions.
> 

Reviewer: ```suggestion
> Gateway API resources are reconciled when the Gateway API CRDs are installed in your cluster _before_  you start the {{ site.kic_product_name }}. See the [getting started](/kubernetes-ingress-controller/{{ page.release }}/get-started/) page for installation instructions.

```
